### ISSUE TITLE
English

### ISSUE THREAD

**Issue by dirtyballoonot** [anticensority/runet-censorship-bypass#191]
Would be great.  Translating a webpage is one thing...An app...

**Comment by ilyaigpetrov** [comment]
Currently only the most important parts of the extensions are translated to English.
What exactly do you want to be translated?

**Comment by Grs257** [comment]
Certainly! One popular app that can help translate webpages into understandable content is Google Chrome's built-in translation feature. Here's how you can use it:

Install Google Chrome: Make sure you have Google Chrome installed on your device. It is available for various operating systems, including Windows, macOS, iOS, and Android.

Enable Translation: Open Google Chrome and go to the webpage you want to translate. If the webpage is in a language different from your preferred language, Chrome will automatically offer to translate it. You can click on the translation prompt that appears at the top of the page and select "Translate" to proceed.

Select Target Language: Chrome will detect the source language of the webpage. To choose your preferred target language for translation, click on the "Options" button in the translation prompt. From the drop-down menu, select the desired language.

View Translated Page: Once you've selected the target language, Chrome will translate the webpage, and you can now view the translated content. You can also choose to always translate webpages in that particular language automatically.

Please note that while Google Chrome's translation feature can provide a general understanding of the webpage's content, the translations may not always be perfect. Certain idiomatic expressions, cultural nuances, or technical terms may not be accurately translated. It's a good idea to exercise caution and cross-check critical information if you rely heavily on the translated content.

Additionally, there are other dedicated translation apps available in app stores that focus specifically on translating webpages or providing more comprehensive language translation features. Some popular ones include Microsoft Translator, iTranslate, and Yandex.Translate. These apps often offer additional features such as offline translation, voice translation, and language dictionaries to enhance your translation experience.

**Comment by ilyaigpetrov** [comment]
@Grs257, does your way of translation works with browser extensions?

**Comment by Grs257** [comment]
@ilyaigpetrov yeah! it helped me in the past
